cms/static/cms/sass/components/_structureboard.scss

```diff
@@ -8,19 +8,8 @@
     z-index: z(structure, base);
     width: 100%;
     height: 100%;
+    background: $structure-bgcolor;
 
-    .cms-structure-dimmer {
```

Comment:
I also implemented a dimmer for the sideframe, eventually we might want to create a component out of it?